adds multiproject ci/cd workshop

ryanmrichard opened the pull request:
This PR resolves #2048 by creating an advertisement for the workshop.

bernhold commented:
@ryanmrichard I'm working on publishing your event (sorry for the delay).  But in checking the links, I've found a couple of things that probably need to be fixed before we publish:

1. Permissions on the registration form are wrong.  I get the following error message from Google when loading it: You need permission. This form can only be viewed by users in the owner's organization.
2. The RFI page (https://multiprojectdevops.github.io/workshop_reports/virtual_workshop1/rfi/) is blank

Please let me know when you have addressed these, or if you'd like to go ahead with publishing the BSSw.io event despite these issues (I don't recommend it).  You should take care of this within the next few days so that we can include your event in the digest that goes out at the end of the month -- that will be the biggest single "advertisement" that we can offer.

ryanmrichard commented:
@bernhold Sorry about that. 

1. I fixed the permissions on the form (and tested with my private account). I should now work. 
2. I tried to get fancy and embed the Google Form for the RFI into the page, but it looks like when it gets hosted on GitHub they won't let that happen. Anyways, I replaced that link with a direct link to the form (and tested permissions with my private account).

bernhold commented:
Hi @ryanmrichard your event is now live: https://bssw.io/events/virtual-workshop-on-multiproject-ci-cd.  Thanks for your contribution!
